Lib/imaplib.py

```diff
@@ -588,6 +648,48 @@ def getquotaroot(self, mailbox):
         return typ, [quotaroot, quota]
 
 
+    def idle(self, dur=None):
```

Comment:
Just noticed this. Can we rename this parameter to `duration`? It's clearer for users to have this be a full word rather than an abbr. (I didn't always think that way but in the current time I think we should try to avoid abbreviated names, especially when the abbr is so arbitrary as `dur` for `duration`, and especially in public interfaces, where the name will be baked into the API. You'll have to fix the docs too.